libyul/AsmParser.h

```diff
@@ -126,7 +126,7 @@ class Parser: public langutil::ParserBase
 	) const;
 
 	/// Creates an inline assembly node with the current source location.
```

Comment:
```suggestion
	/// Creates an inline assembly node with the current debug data.
```
I feel like Kamil :-D

Reply:
You did change this before, didn't you :-)? But now it seems like you lost it (in rebase or something) again?